api-spec/api-spec.md

```diff
@@ -1,43 +1,45 @@
 
-In WFS 3 Collections are the sets of data that can be queried ([7.11](https://rawgit.com/opengeospatial/WFS_FES/master/docs/17-069.html#_feature_collections_metadata)), and each describes basic information about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all the data contained. [STAC collections](../collections-spec/README.md) contain this same information, along with other STAC specific fields and thus are compliant with both WFS Collections and STAC Collections and is returned from the `/collections/{collection_id}` endpoint.
+In WFS 3 Collections are the sets of data that can be queried ([7.11](https://rawgit.com/opengeospatial/WFS_FES/master/docs/17-069.html#_feature_collections_metadata)), and each describes basic information about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all the data contained. [STAC collections](../collection-spec/README.md) contain this same information, along with other STAC specific fields and thus are compliant with both WFS Collections and STAC Collections and is returned from the `/collections/{collection_id}` endpoint.
+
+In WFS 3 Features are the individual records within a Collection and are provided in GeoJSON format. [STAC Items](../item-spec/README.md) are analogous to WFS 3 Features, are in GeoJSON, and are returned from the `/collections/{collection_id}/items/{item_id}` endpoint.
+
+## HTTP Request Methods and Content Types
 
-In WFS 3 Features are the individual records within a Collection and are provided in GeoJSON format. [STAC Items](../item-spec/README.md) are analagous to WFS 3 Features, are in GeoJSON, and are returned from the `/collections/{collection_id}/items/{item_id}` endpoint.
+For WFS3-compliance, it is only **required** that `GET` be implemented for the WFS3 endpoints.  It is also only **required** that `GET` be implemented for the STAC endpoints. 
 
-### WFS3 Endpoints
+Since STAC adds additional filter parameters that may have much larger values, like `intersects`, it is **recommended** to also support `POST` for both the WFS3 and STAC endpoints that
+ accept filter parameters (e.g., `/collections/{collection_id}/items` and `/stac/search`). 
+ 
+It is **recommended** that these `Content-Type` options be supported when using POST:
 
-The core WFS 3 endpoints are shown below, with details provided in an [OpenAPI specification document](definitions/WFS3.yaml).
+1. POST with the `Content-Type` header set to `application/x-www-form-urlencoded` with the appropriate content body format.
```

Comment:
Is there an easy way to transform between the three? Otherwise this feels maybe a bit much to ask for.

Reply:
same for my Scala/Akka implementation -- just a few lines of code to handle all three cases.

Python Flask example: https://www.flaskapi.org/api-guide/parsers/

Reply:
Okay, great.